README.adoc

```diff
@@ -0,0 +1,103 @@
+own, service developers can focus on the unique features of their services.
+
+*Repose* is run as a standalone application (either as a Linux service or using the
+JAR directly). *Repose* can be run on the same server or on a different server, and
+it can be run across multiple servers for horizontal scaling. At its core, *Repose* is a
+proxy that allows services to use Enterprise Integration Patterns (EIP).
+
+For more information, check out our http://www.openrepose.org/versions/latest/recipes/getting-started.html[Getting Started with *Repose*] guide.
+
+
+== Benefits
+
+* *Scalable*. *Repose* is incredibly scalable because it is designed to be stateless.
+* *Flexible*. *Repose* is run as a http://www.openrepose.org/versions/latest/recipes/valve-installation.html[standalone Linux service (Valve)].
+* *Extensible*. New http://www.openrepose.org/versions/latest/filters/index.html[components]
+  are being added all of the time, and you can even build your own
+  https://github.com/rackerlabs/repose-hello-world[custom component].
+* *High performance*. *Repose* can handle http://www.openrepose.org/versions/latest/recipes/performance-best-practices.html[high loads] with high accuracy.
+* *Improving*. *Repose* continues to be under https://github.com/rackerlabs/repose/releases[active development].
+
+
+== *Repose* Components
+
+*Repose* includes several http://www.openrepose.org/versions/latest/filters/index.html[filters] and http://www.openrepose.org/versions/latest/services/index.html[services]
+out of the box.  These include:
+
+* http://www.openrepose.org/versions/9.0.0.0/filters/rate-limiting.html[Rate Limiting]
+* http://www.openrepose.org/versions/9.0.0.0/filters/keystone-v2.html[Keystone v2 Auth]
+* http://www.openrepose.org/versions/9.0.0.0/filters/openstack-identity-v3.html[OpenStack v3 Auth]
+* http://www.openrepose.org/versions/9.0.0.0/filters/api-validator.html[API Validation]
+* http://www.openrepose.org/versions/9.0.0.0/filters/translation.html[Translation]
+* http://www.openrepose.org/versions/9.0.0.0/filters/compression.html[Compression]
+* http://www.openrepose.org/versions/9.0.0.0/filters/cors.html[CORS]
+* http://www.openrepose.org/versions/9.0.0.0/filters/versioning.html[Versioning]
+* http://www.openrepose.org/versions/9.0.0.0/filters/slf4j-http-logging.html[HTTP Logging]
+
+*Repose* also makes it easy to create your own custom components.  Check out
+our https://github.com/rackerlabs/repose-hello-world[example custom filter] for more details.
+
+
+== Installation
+You can install *Repose* using the following methods:
+
+* http://www.openrepose.org/versions/9.0.0.0/recipes/valve-installation.html#yum-based-linux-distributions[yum]
+* http://www.openrepose.org/versions/9.0.0.0/recipes/valve-installation.html#apt-based-linux-distributions[apt]
+* http://www.openrepose.org/versions/9.0.0.0/recipes/valve-installation.html#yum-based-linux-distributions[Docker]
```

Comment:
This link should probably be to the Docker page?
http://www.openrepose.org/versions/9.0.0.0/recipes/docker.html

Reply:
Also, for these links and the ones above, would we want to use the `latest` version rather than `9.0.0.0`? So that we don't have to update this README every release.

Reply:
yeah, some how my copy didn't take on that one. and i meant to do a find replace when i was done

Reply:
i fix